packages/parse5-test-utils/data/parser-feedback/entities01.test

```diff
@@ -60,6 +60,26 @@
                 ]
             ]
         },
+        {
```

Comment:
Could this one be added in a separate PR, to emphasize that something important was fixed?

Reply:
This was generated by the parser feedback generation script. Nothing fixed here, it just seems like that script wasn't run for a while and had some catching up to do.

Reply:
As this isn't linked to any code changes, I'm open to reverting this. There were some actually breaking changes generated by the script, which I simply didn't include for now.

Reply:
Yeah, :+1: on that

Reply:
Done